reflex/testing.py

```diff
@@ -397,7 +397,23 @@ def frontend(self, driver_clz: Optional[Type["WebDriver"]] = None) -> "WebDriver
             )
         if self.frontend_url is None:
             raise RuntimeError("Frontend is not running.")
-        driver = driver_clz() if driver_clz is not None else webdriver.Chrome()
+        want_headless = False
+        options = None
+        if os.environ.get("APP_HARNESS_HEADLESS"):
```

Comment:
Do we start adding the `REFLEX_` prefix now for the env vars, or do we use a different PR to change all of them?

Reply:
Let's have a separate PR where we do this for all the env vars